libraries/AP_Scripting/drivers/LTE_modem.lua

```diff
@@ -148,49 +195,294 @@ local log_file = io.open('SIM7600.log', 'w')
 --]]
 local function uart_read()
     local s = uart:readstring(512)
-    if s and #s > 0 and log_file then
-        log_file:write('[' .. s .. ']\n')
-        log_file:flush()
-    end
+    log_data(s, '<<<')
+    stats.bytes_in = stats.bytes_in + #s
     return s
 end
 
+--[[
+    Function to write to the UART and log the command
+--]]
+local function uart_write(s)
```

Comment:
BTW, you can use the same markup as the lua docs to get better type checking. Once scripts start getting complex its worth the effort. 

For example:
```suggestion
---@param s string
---@return number
local function uart_write(s)
```

Reply:
ok, thanks for the note